x-pack/solutions/security/packages/ecs-data-quality-dashboard/impl/data_quality_panel/data_quality_details/indices_details/pattern/index_check_flyout/incompatible_tab/index.tsx

```diff
@@ -95,6 +98,10 @@ const IncompatibleTabComponent: React.FC<Props> = ({
     ]
   );
 
+  const chatTitle = useMemo(() => {
+    return `${indexName} - ${getFormattedCheckTime(checkedAt)}`;
+  }, [checkedAt, indexName]);
+
```

Comment:
As I mentioned  in https://github.com/elastic/kibana/pull/211831/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&file-filters%5B%5D=.yaml&owned-by%5B%5D=kapral18&show-deleted-files=true&show-viewed-files=true#r1969992177 this calculation should also probably happen for legacycheck fields so this title should probably be lifted up. And the getFormattedCheckTime can be passed down from `IndexCheckFlyout`